### ISSUE TITLE
two monitors 

### ISSUE THREAD

**Issue by hubertsvk** [phaselden/FlipIt#42]
Hello,
it is possilbe to select two monitors to diplay clock?

**Comment by phaselden** [comment]
Yes. And you can configure what appears on each. I have a large monitor attached to my laptop. I should the normal flip clock on my laptop monitor and the world times on my large monitor
